frontend/src/router/index.ts

```diff
@@ -13,6 +13,11 @@ const router = createRouter({
       name: 'LogIn',
       component: import('../views/LogInView.vue'),
     },
+    {
+      path: '/sequence',
```

Comment:
we could redirect to this page after login (just change the line 98 in the loginview)